Formula/protoc-gen-go.rb

```diff
@@ -1,11 +1,15 @@
 class ProtocGenGo < Formula
   desc "Go support for Google's protocol buffers"
-  homepage "https://github.com/golang/protobuf"
-  url "https://github.com/golang/protobuf/archive/v1.5.2.tar.gz"
-  sha256 "088cc0f3ba18fb8f9d00319568ff0af5a06d8925a6e6cb983bb837b4efb703b3"
+  homepage "https://github.com/protocolbuffers/protobuf-go"
+  url "https://github.com/protocolbuffers/protobuf-go/archive/v1.26.0.tar.gz"
+  sha256 "26218474bcf776ecf32d7d194c6bfaca8e7b4f0c087e5b595fd50fbb31409676"
   license "BSD-3-Clause"
-  revision 1
-  head "https://github.com/golang/protobuf.git"
+  head "https://github.com/protocolbuffers/protobuf-go.git"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
```

Comment:
Why do we need `:github_latest`? We should only use this when we know it's needed.

Reply:
Just copy from another formula. 😂

Reply:
Then if we don't know that we need it, let's not put this in.